ballerina/client.bal

```diff
@@ -63,7 +63,7 @@ public isolated client class Client {
     # + oauth_token - OAuth 2.0 token for the current user.
     # + prettyPrint - Returns response with indentations and line breaks.
     # + quotaUser - An opaque string that represents a user for quota purposes. Must not exceed 40 characters.
-    # + return - A `calendar:Calendar` if successful, otherwise an `calendar:Error` 
+    # + return - A `calendar:Calendar` if successful, otherwise an `calendar:Error`
```

Comment:
```suggestion
    # + return - A `calendar:Calendar` if successful, otherwise a `calendar:Error`
```

Reply:
Fixed in [0a3043d](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/138/commits/0a3043d4f58f8506633f1d24410035c1ec305f87)